gating/generate_release_notes/release_notes_dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:16.04
+RUN apt-get update && apt-get install -y python-pip build-essential python-dev libssl-dev
+RUN apt-get install -y libffi-dev
+RUN apt-get install -y sudo
+RUN apt-get install -y git-core
+RUN useradd jenkins --shell /bin/bash --create-home --uid 500
+RUN echo "jenkins ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
+RUN apt-get install -y pandoc
+
+# use released versions of the differs when they are available.
+RUN pip install rpc_differ reno
```

Comment:
If this can't be made to work with a constraints file, I think it would be worth pinning them here.

Reply:
Added pins.